cmd/kubectl-gadget/deploy.go

```diff
@@ -64,16 +64,19 @@ var deployCmd = &cobra.Command{
 var gadgetimage = "undefined"
 
 var (
-	image               string
-	imagePullPolicy     string
-	hookMode            string
-	livenessProbe       bool
-	deployTimeout       time.Duration
-	fallbackPodInformer bool
-	printOnly           bool
-	quiet               bool
-	debug               bool
-	wait                bool
+	image                string
+	imagePullPolicy      string
+	hookMode             string
+	livenessProbe        bool
+	deployTimeout        time.Duration
+	fallbackPodInformer  bool
+	printOnly            bool
+	quiet                bool
+	debug                bool
+	wait                 bool
+	socketPathContainerd string
+	socketPathCRIO       string
+	socketPathDocker     string
```

Comment:
> Since, we can only select one container runtime
>
Actually, in a multi-node cluster, we could have different runtimes on each node. So, if we want to manage such a case, we need all three flags, even if it makes the CLI more complex 🙁

Reply:
They will increase if we add support for other runtimes, so I think that's a price to pay for making IG project compatible with more systems out there 🙂